puppet/modules/jenkins_job_builder/files/theforeman.org/scripts/pr_checkout.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -xe
+
+cd plugin/
```

Comment:
I think separate's probably going to be best, or it needs to take a variable.. it's probably best not to assume a certain layout, or what type of PR we're testing.  (Perhaps it's testing a non-plugin PR against the plugin.)